server/spike/adk/spike_08_subagents.py

```
@@ -0,0 +1,210 @@
+                    print(f"  [{elapsed}s] [{agent_name}] TOOL_RESULT: {part.function_response.name}")
+                elif part.text:
+                    text_response += part.text
+                    events_by_agent[agent_name].append(("text", len(part.text)))
+                    preview = part.text[:80].replace("\n", " ")
+                    print(f"  [{elapsed}s] [{agent_name}] TEXT: {preview}...")
+
+    wall_time = round(time.time() - start, 2)
+
+    # ── Validation ────────────────────────────────────────────────────────────
+
+    print(f"\n── Results ──")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)